easybuild/easyconfigs/f/freetype/freetype-2.6.3-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,50 @@
+# Built with EasyBuild version 3.3.1 on 2018-01-03_12-06-31
+#
+name = 'freetype'
+version = '2.6.3'
+
+homepage = 'http://freetype.org'
+description = """FreeType 2 is a software font engine that is designed to be small, efficient, highly customizable, and
+ portable while capable of producing high-quality output (glyph images). It can be used in graphics libraries, display
+ servers, font conversion tools, text image generation tools, and many other products as well."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+
+source_urls = [GNU_SAVANNAH_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('libpng', '1.6.23')]
```

Comment:
libpng 1.6.30 is already available on daint

Reply:
done.